lib/ansible/modules/cloud/docker/docker_node_facts.py

```diff
@@ -52,31 +64,40 @@
 '''
 
 EXAMPLES = '''
+- name: Get info on all nodes
+  docker_node_facts:
+  register: result
+
 - name: Get info on node
   docker_node_facts:
     name: mynode
   register: result
 
+- name: Get info on list of nodes
+  docker_node_facts:
+    name:
+      - mynode1
+      - mynode2
+  register: result
+
+- name: Get info on host if it is Swarm Manager
+  docker_node_facts:
+    self: true
+  register: result
 '''
 
 RETURN = '''
-exists:
+nodes_facts:
     description:
-      - Returns whether the node exists in docker swarm cluster.
-    type: bool
+      - Facts representing the current state of the nodes. Matches the C(docker node inspect) output.
+      - Can contain multiple entries if more than one node provided in I(name), or I(name) is not provided.
+      - If I(name) contains a list of nodes, the output will contain information on all nodes registered at the swarm,
+        including nodes that left the swarm but hasn't been removed from the cluster on swarm managers and nodes that
```

Comment:
I think the new version is much better!
```suggestion
        including nodes that left the swarm but haven't been removed from the cluster on swarm managers and nodes that
```